src/Orleans/Serialization/BuiltInTypes.cs

```diff
@@ -1836,8 +1836,10 @@ internal static void SerializeInvokeMethodRequest(object obj, ISerializationCont
             var request = (InvokeMethodRequest)obj;
 
             context.StreamWriter.Write(request.InterfaceId);
+            context.StreamWriter.Write(request.InterfaceVersion);
```

Comment:
InvokeMethodRequest should in theory only be used for messaging, and not persistence. So do we care if it's not back compat, since the entire change itself is not backcompat to for side by side deployment?